zeebe/gateway-protocol/src/main/proto/rest-api.yaml

```diff
@@ -1706,26 +1721,45 @@ components:
           format: int64
         tenantId:
           type: string
-    AuthorizationAssignRequest:
+    AuthorizationPatchRequest:
       type: object
       properties:
-        ownerKey:
-          type: integer
-          format: int64
-        ownerType:
+        action:
+          description: Indicates if permissions should be added or removed.
           type: string
           enum:
+            - ADD
+            - REMOVE
+        resourceType:
+          description: The type of resource to add/remove perissions to/from.
+          enum:
+            - AUTHORIZATION
```

Comment:
❓ I like the addition of this as an enum, its a known set of values from us so its suited to an enum, my question is if I think about how authorizations are applied at a processor level, does this match with what we would use (ValueType) to determine the authorizations to fetch?

Reply:
Most of them would match a ValueType, but not all. For example, we don't have an `APPLICATION` ValueType. All the others would match. I'd also argue `USER_GROUP` could be called just `GROUP`. But for now I just took what was listed in the document.

Reply:
Thats ok with me, I just wanted to check if it impacted how the authorizations are applied lower down, but you are aware of that functionality so I will leave it to you :)